apps/builder/app/builder/features/sidebar-left/sidebar-left.tsx

```diff
@@ -59,6 +61,20 @@ export const SidebarLeft = ({ publish }: SidebarLeftProps) => {
             </SidebarTabsTrigger>
           ))}
         </SidebarTabsList>
+        <Box css={{ borderRight: `1px solid  ${theme.colors.borderMain}` }}>
+          <SidebarTabsTrigger
+            as={"button"}
```

Comment:
Just curious wny not link with `target="_blank"
As in case of something really bad with js, native link will work

Reply:
yep, problem was types of that button don't take an anchor and I didn't want to rewrite the button